bridge/image.yaml

```diff
@@ -65,10 +65,7 @@ packages:
   remove:
     - vim-minimal
     - subscription-manager
-    - shadow-utils
```

Comment:
Is shadow-utils actually required runtime?   In another place, @scholzj indicated that shadow-utils is only required upstream to add the strimzi user to the passwd db.  We don't need that downstream.

Reply:
I haven't run any tests ... but I suspect we install it because we add the Kafka user in our Dockerfiles:
```
RUN useradd -r -m -u 1001 -g 0 kafka
```
But I don't think we use anything fro that at runtime.

Reply:
shadow-utils removed